Expanded the bug reporting guidelines

Mailaender opened the pull request:
A humble wish.

Mailaender committed:
Update CONTRIBUTING.md

please don't submit duplicates and keep it one issue per issue

penev92 commented on CONTRIBUTING.md:
```diff
@@ -5,6 +5,8 @@
 * Add the appropriate log files on crashes.
 * Please be specific on how to reproduce the problem.
+* Do not submit duplicate issues. Search first.
+* Avoid including few requests/bugs into a single issue.
```
"few" -> "several"

penev92 commented:
:+1: although I doubt most people read those. :/

Phrohdoh commented:
s/few/multiple

clemty commented:
Perhaps tell people to say which version they are using? (to avoid reports for long-fixed bugs in bleed because people use the last release)

Mailaender commented:
Updated.

> Perhaps tell people to say which version they are using?

The situation of not mentioning the version has improved since https://github.com/OpenRA/OpenRA/pull/3938, but nonetheless that is important and often forgotten and a problem when no logs are provided. Added.

obrakmann commented:
:+1:
